### PR TITLE
Add Amenities Checkboxes and update JSON

### PR DESCRIPTION
**Pull request by emejia01**
Added checkboxes for amenities to the UI and connected the inputs to the Listings JSON object. Also started initial logic to implement an empty state for the MyListingsPage.

### CODE REVIEW

File: vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue [hunk 1]
```diff
@@ -200,10 +187,18 @@ export default {
       'Washington', 'West Virginia', 'Wisconsin', 'Wyoming',
     ],
   }),
+
+  computed: {
```
Comment (NicolasCapparelli) [inline vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue:191]: Curious, why is this a computed property if it's just returning a boolean literal?

File: vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue [hunk 2]
```diff
@@ -230,28 +236,51 @@ export default {
...
+
       let response = await fetch('/locations', {
         method: "POST",
         headers: {
           "Content-Type": "application/json"
         },
         body: JSON.stringify(currentListing)
       })
+      
+      console.log(totalBeds);
```
Comment (NicolasCapparelli) [inline vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue:268]: nit: remove console logs so they don't end up in prod

File: vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue [hunk 3]
```diff
@@ -213,14 +208,25 @@ export default {
...
-      
+      // Create Listing Object
       const currentListing = {
         name: listingTitle,
         price: listingPrice,
         type: propertyType,
         desc: listingDescription,
+
         images: [
           "https://cdngeneral.rentcafe.com/dmslivecafe/3/1104500/METRO%20GATEWAY%20IMG%2003(2).jpg?crop=(0,0,300,191.25000000000028)&cropxunits=300&cropyunits=200&quality=85&scale=both",
```
Comment (qirongwang) [inline vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue:231]: where are these images from?

Comment (emejia01) [reply vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue:231]: Those are temp images for now (placeholders)

**Pull request merged**